labgrid/remote/coordinator.py

```diff
@@ -653,12 +653,19 @@ async def acquire_place(self, name, details=None):
         return True
 
     @locked
-    async def release_place(self, name, details=None):
+    async def release_place(self, name, acquired, details=None):
```

Comment:
Does this break compatibility with older clients?

Reply:
Yes, but we can catch this by assigning `acquired=None`.

Reply:
Does that fix the other combination as well?

Reply:
which other combination?